portality/tasks/journal_in_out_doaj.py

```diff
@@ -24,23 +34,23 @@ def run(self):
         job = self.background_job
         params = job.params
 
-        journal_id = params.get("set_in_doaj__journal_id")
+        journal_ids = params.get("set_in_doaj__journal_ids")
         in_doaj = params.get("set_in_doaj__in_doaj")
 
-        if journal_id is None or in_doaj is None:
+        if journal_ids is None or len(journal_ids) == 0 or in_doaj is None:
```

Comment:
If journal_ids is actually none as opposed to [], this line will crash with `TypeError: object of type 'NoneType' has no len()`. Perhaps simply "if journal_ids or in_doaj is None"? (you can't do the same to in_doaj since it could legitimately be False, but a list should only ever be [] or have some items)